packages/falso/src/lib/float.ts

```diff
@@ -20,10 +20,22 @@ export interface RandomFloatOptions extends RandomInRangeOptions, FakeOptions {}
  *
  * randFloat({ length: 10 })
  *
+ * @example
+ *
+ * randFloat({ min: 10, max: 20, fraction: 1 })
+ * // 12.5
```

Comment:
Can you please run the docs and see that it looks OK with the comment?

Reply:
Looks good
<img width="708" alt="Screenshot 2022-03-22 at 09 29 33" src="https://user-images.githubusercontent.com/13211/159439151-684b2e2f-cb04-449b-8232-98089e6e9359.png">

Or would you rather have it one line?

Reply:
I meant in the live docs website :)

Reply:
Ahhh :)

I think it will look better if I have it on the same line, I'll change it

<img width="537" alt="Screenshot 2022-03-22 at 10 11 35" src="https://user-images.githubusercontent.com/13211/159446311-8c0eb4de-7ab5-4ed4-b575-d6a52638e3b6.png">